### ISSUE TITLE
training dataset  DNSMOS

### ISSUE THREAD

**Issue by xxoospring** [microsoft/DNS-Challenge#157]
Has the dataset downloaded by the DNS4 script been filtered by DNSMOS>4.25（Track1） as the DNS4 paper said in section3.1? I checked some data, for example in spanish_speech/SLR39_48kHz, and found that these data have low DNSMOS values (calculated by dnsmos_local.py). So, should I filter these data out when building my training set?

**Comment by vishakg** [comment]
I don't believe that we did that filtering for Spanish_speech. @hadubey can you confirm?

**Comment by xxoospring** [comment]
> I don't believe that we did that filtering for Spanish_speech. @hadubey can you confirm?
And what about the others, besides Spain?
